mainline/stretch/Dockerfile

```
@@ -89,7 +89,8 @@ RUN set -x \
 	fi
 
 # implement changes required to run NGINX as an unprivileged user
-RUN chown -R :0 /var/cache/nginx \
+RUN usermod -u 1001 nginx \
+	chown -R 1001:0 /var/cache/nginx \
```

Missing the leading `&&`?
```suggestion
	&& chown -R 1001:0 /var/cache/nginx \
```

---

Well, this is a new GitHub feature?